DICOM/src/Microsoft.Health.DeID.SharedLib.UnitTests/Microsoft.Health.Anonymizer.Common.UnitTests.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
```

Comment:
🔴 [Insist]

This file is not needed, right? At least the path has `DeID` word. 🤔

Reply:
deleted it, thanks